RHEL/6/input/checks/avahi_disable_publishing.xml

```diff
@@ -0,0 +1,28 @@
+<def-group>
+  <definition class="compliance" id="avahi_disable_publishing" version="1">
+    <metadata>
+      <title>Disable Avahi Publishing</title>
+      <affected family="unix">
+        <platform>Red Hat Enterprise Linux 6</platform>
+      </affected>
+      <description>Check that Avahi is ignoring packets
+      unless the TTL field is 255.
+      </description>
+      <reference source="galford" ref_id="20150316" ref_url="test_attestation" />
+    </metadata>
+    <criteria comment="Disable Avahi Publishing" operator="OR">
+      <extend_definition comment="avahi package is not installed" definition_ref="package_avahi_installed" negate="true" />
+      <criterion comment="Avahi publishing is disabled" test_ref="test_avahi_disable_publishing" />
+    </criteria>
+  </definition>
+
+  <ind:textfilecontent54_test check="all" check_existence="all_exist" comment="Avahi TTL setting" id="test_avahi_disable_publishing" version="1">
```

Comment:
Update comments (not to speak about 'Avahi TTL setting' but rather about disabling publishing'). Here and at line 22 in the TFC_O too. Update pattern not to allow spaces around '=' character. Otherwise looks fine.